api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
Params themselves don't seem to show up when viewing through an anonymous link, however? The contributor information would be under params.  Is this sufficient, or do I definitely need the UserSerializer?


![screen shot 2016-04-22 at 10 24 48 am](https://cloud.githubusercontent.com/assets/9755598/14744344/6e509bd2-0874-11e6-96ea-7d407d74ec44.png)